test/suites/standard1/router.robot

```diff
@@ -192,3 +223,21 @@ Setup Hello MicroShift Pods In Multiple Namespaces
     Expose Hello MicroShift    ${NS_OWNERSHIP_2}
     Oc Expose    svc hello-microshift --hostname ${HOSTNAME} --path /${NS_OWNERSHIP_1} -n ${NS_OWNERSHIP_1}
     Oc Expose    svc hello-microshift --hostname ${HOSTNAME} --path /${NS_OWNERSHIP_2} -n ${NS_OWNERSHIP_2}
+
+Http Port Should Be Open
+    [Documentation]    Try to curl the router, disregard the response because we only care about connectivity.
+    [Arguments]    ${port}
+    ${result}=    Access Hello MicroShift    ${port}
+    Should Match Regexp    ${result}    HTTP.*503
```

Comment:
Why do we expect a 503?

Reply:
Added a comment. This is because there is no route in the router to handle this path. The test only checks connectivity and a router listening on the other side, not its configuration.